src/media/css/addon-tiles.styl

```diff
@@ -0,0 +1,27 @@
+@import 'lib';
+
+.addon-header .mkt-tile-info {
```

Comment:
for addons, i'd get rid of any class name that starts with ```.mkt``` to decouple it from app styles